diff --git a/android/build.gradle b/android/build.gradle
index 6a41eb001ddcf93a07ce97766dc0aaab7e937359..5328ed1531903e0a663b4497f8289ecf8b3934dc 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -4,7 +4,7 @@ buildscript {
 
   repositories {
     google()
-    jcenter()
+    mavenCentral()
   }
 
   dependencies {
@@ -52,7 +52,6 @@ android {
 
 repositories {
   mavenCentral()
-  jcenter()
   google()
 
   def found = false
